packages/terra-signature/src/Signature.jsx

```diff
@@ -0,0 +1,204 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import 'terra-base/lib/baseStyles';
+import './Signature.scss';
+
+const LineWidths = {
+  EXTRAFINE: 1,
+  FINE: 2,
+  MEDIUM: 4,
+  HEAVY: 6,
+};
+
+
+const propTypes = {
+  /**
+   * The line width to use when drawing the signature on the canvas.
+   * One of LineWidths.EXTRAFINE, LineWidths.FINE, LineWidths.MEDIUM, LineWidths.HEAVY.
+   */
+  lineWidth: PropTypes.oneOf([LineWidths.EXTRAFINE, LineWidths.FINE, LineWidths.MEDIUM, LineWidths.HEAVY]),
+
+  /**
+  * Line segments that define signature.
+  */
+  lineSegments: PropTypes.array,
+
+  /**
+   * A callback function to execute when a line segment is drawn. The first parameter is the event, the
+   * second parameter is all the line segments, and the last parameter is the most recent line segment drawn.
+   */
+  onChange: PropTypes.func,
+
```

Comment:
Nit: It might not be worth updating. I don't think there is an official standard for the propTypes comment spacing, but looking at a few other examples it seems like there is normally no spaces between comments.

https://github.com/cerner/terra-core/blob/master/packages/terra-alert/src/Alert.jsx#L26-L65
https://github.com/cerner/terra-core/blob/master/packages/terra-button/src/Button.jsx#L9-L58